wallet/how-to/use-3rd-party-integrations/js-infura-api.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 2
+description: Use Infura and custom nodes to make direct, read-only requests.
+---
+
+# Make read-only requests from your JavaScript dapp
+
+You can use the [Infura API](https://docs.infura.io/) from your JavaScript dapp with
+[MetaMask SDK](../connect/set-up-sdk/javascript/index.md) installed to make direct, read-only
+JSON-RPC requests.
+
+Direct, read-only JSON-RPC requests are blockchain requests that do not require user wallet interaction.
+Your dapp can directly call most [JSON-RPC API methods](/wallet/reference/eth_subscribe), bypassing
+user wallet authentication for read-only operations.
+
+<details>
```

Comment:
This seems like important information, but it's hidden by default. Perhaps add in a normal paragraph?